docs-guides/source/model-prediction.md

```diff
@@ -160,3 +160,140 @@ img_as_np_array = np.reshape(img_as_np_array, model_expected_input_shape)
 out_dict = model.predict({'image': img_as_np_array})
 ```
 
+
+## Using Compiled Python Models for Prediction
+
+You can use a *compiled* Core ML model ([CompiledMLModel](https://apple.github.io/coremltools/source/coremltools.models.html#coremltools.models.CompiledMLModel)) rather than  [MLModel](https://apple.github.io/coremltools/source/coremltools.models.html#coremltools.models.model.MLModel) for making predictions. For large models, using a compiled model can save considerable time in initializing the model. 
+
+For example, [Stable Diffusion](https://machinelearning.apple.com/research/stable-diffusion-coreml-apple-silicon), adopted by a vibrant community of artists and developers, enables the creation of unprecedented visuals from a text prompt. When using [Core ML Stable Diffusion](https://github.com/apple/ml-stable-diffusion#core-ml-stable-diffusion), you can speed up the load time after the initial load by first copying and storing the location of the `mlmodelc` compiled model to a fixed location, and then initializing the model from that location. 
+
+```{note}
+You can't modify a compiled model like you can an [MLModel](https://apple.github.io/coremltools/source/coremltools.models.html#coremltools.models.model.MLModel) loaded from a non-compiled `mlpackage` model file. 
+```
+
+### Why Use a Compiled Model?
+
+When you initialize a model using (in Python) `model ct.models.MLModel("model.mlpackge")`, the Core ML Framework is invoked and the following steps occur, as shown in the following diagram.
+
+```{figure} images/model-lifecycle.png
+:alt: Initialize MLModel
+:align: center
+:class: imgnoborder
+
+This diagram is from [Improve Core ML integration with async prediction](https://developer.apple.com/videos/play/wwdc2023/10049/),
+presented at the Apple 2023 World Wide Developer Conference.
+```
+
+1. The `mlpackage` is [compiled](https://developer.apple.com/documentation/coreml/mlmodel/3929553-compilemodelaturl) into a file with extension `mlmodelc` . This step is usually very fast.
+
+2. The compiled model is then [instantiated](https://developer.apple.com/documentation/coreml/mlmodel/3022229-modelwithcontentsofurl) using the specified `compute_units`, and captured in the MLModelConfiguration config. 
```

Comment:
--> `using the specified compute_units, captured in the MLModelConfiguration config. `

Reply:
OK but no need for comma.